CMakeLists.txt

```diff
@@ -44,38 +50,59 @@ configure_file(
     @ONLY
 )
 
-set(CMAKE_CXX_EXTENSIONS OFF)
-
-include(FetchContent)
-include(GNUInstallDirs)
 
 # [CMAKE.LIBRARY_NAME]
 add_library(beman.any_view INTERFACE)
 # [CMAKE.LIBRARY_ALIAS]
 add_library(beman::any_view ALIAS beman.any_view)
-target_include_directories(
+
+set_target_properties(
     beman.any_view
-    INTERFACE
-        $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
-        $<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/include>
-        $<INSTALL_INTERFACE:include>
+    PROPERTIES
+        VERIFY_INTERFACE_HEADER_SETS ON
+        EXPORT_NAME any_view
 )
 
-install(
-    TARGETS beman.any_view
-    EXPORT ${TARGETS_EXPORT_NAME}
-    DESTINATION
-    $<$<CONFIG:Debug>:debug/>${CMAKE_INSTALL_LIBDIR}
+target_sources(
+    beman.any_view
+    PUBLIC
+        FILE_SET HEADERS
+        BASE_DIRS
+            ${CMAKE_BINARY_DIR}/include
+            include
+        FILES
+            ${CMAKE_BINARY_DIR}/include/beman/any_view/config.hpp
```

Comment:
Why `CMAKE_BINARY_DIR` instead of `PROJECT_BINARY_DIR`?

Reply:
This should probably be `CMAKE_CURRENT_BINARY_DIR` actually, will fix

The reason is because that's where `configure_file()` is going to put its output:

>  relative path is treated with respect to the value of CMAKE_CURRENT_BINARY_DIR

Reply:
Fixed